src/examples/libpmem2/redo/redo.c

```diff
@@ -420,14 +420,16 @@ main(int argc, char *argv[])
 	}
 	fd = open(path, O_RDWR);
 	if (fd < 0) {
-		perror("open");
-		return 1;
+		ret = 1;
+		goto err_open;
```

Comment:
I do not understand this change. you are removing errormsg and replacing return with goto when it's not needed. (and we have more returns like this in code above )

Reply:
This is the only place where the error message was printed, so don't think it was useful. I changed this so that errors are handled the same way everywhere.

tbh, idc how this is changed...

Reply:
we print errormgs everywhere (please see pool_map)

Reply:
done